spec_2/scrabble.js

```diff
@@ -0,0 +1,57 @@
+const playedTiles = "AERETOXMYCNS_B"
```

Comment:
You have a series of related functions and variables, with one overarching purpose: create the `showRemaningTiles()` function. 

In the future, might be worth grouping them into an object to demonstrate their cohesive purpose.

This would allow me, as a user / reviewer, to run `showRemainingTiles` using a variety of inputs.